src/faq.md

```diff
@@ -1,30 +1,30 @@
 
 <br/>
 
-**Q:** How do I know you're not looking at my passwords?:
+**Q:** How do I know you're not looking at my passwords?
 
 **A:** When you type, your computer takes the app name and your master password and using them makes a new password. Nothing is ever sent across the internet, other than the code for the website itself. If you really want, you can [download the files](https://github.com/cloverleaf/web/archive/refs/heads/master.zip) for the website, smash your router and run Cloverleaf offline (Not that I'd recommend smashing your router.)
 
 <br/>
 
-<a name="remake">**Q:** Why did you remake an existing product?:</a>
+<a name="remake">**Q:** Why did you remake an existing product?</a>
 
-**A:** I've openly acknowledged that Cloverleaf is based off the concept of "[Master Password](https://masterpasswordapp.com/)" but why would I spend all this time re-making a working piece of software? The answer is simple: *I don't like how Master Password works.* It has many different offshoots that all look different. It takes 6 variables to make a password and 30 seconds to load (Yes, I timed it.). It makes everything harder than it should be. And that was my motivation to make Cloverleaf. I absolutely love the concept of secure passwords without storing them but hate the current solutions.
+**A:** I've openly acknowledged that Cloverleaf is based off the concept of "[Master Password](https://masterpasswordapp.com/)" but why would I spend all this time re-making a working piece of software? The answer is simple: *I don't like how Master Password works.* It has many different offshoots that all look different. It takes 6 variables to make a password and 30 seconds to load (Yes, I timed it.). It makes everything harder than it should be. And that was my motivation to make Cloverleaf. I absolutely love the concept of secure passwords without storing them but dislike the current solutions.
 
 <br/>
 
-**Q:** What if I have multiple accounts on one service?:
+**Q:** What if I have multiple accounts on one service?
 
 **A:** There isn't a system in place for this but there doesn't need to be since you can just type `ACCOUNTNAME PASSWORD` in the password field to get the password for that account.
 
 <br/>
 
-<a name="cookies">**Q:** What are cookies and why do you use them?:</a>
+<a name="cookies">**Q:** What are cookies and why do you use them?</a>
 
 **A:** A cookie is a small bit of information that a website stores on your computer so that when you come back to the site later, it can remember things from your previous visit. Cloverleaf uses these for remembering what theme you like and a few other things like that.

```

Comment:
Would be good to show what exactly the cookies are used for, with links to where they are defined in the codebase.